spec/integration/README.md

```diff
@@ -94,20 +93,24 @@ Run `$ rake -T` to show all the available rake tasks.
 Then run any number of integration tests:
 
 ```ruby
-$ rake spec:integration
-$ rake spec:integration:create
-$ rake spec:integration:update
-$ rake spec:integration:delete
-$ rake spec:integration:api_version[ver]        # eg, spec:integration:api_version[300]
-$ rake spec:integration:api[version,model]      # eg, spec:integration:api_version[300,c7000]
-$ rake spec:integration:delete:api_version[ver] # eg, spec:integration:delete:api_version[300]
-$ rake test:path[path]        # eg, test:path['spec/integration/resource/api200/connection_template']
+$ rake integration
+$ rake integration:path[path]  # e.g. integration:path[spec/integration/resource/api200/connection_template]
+
+$ rake integration:only[action,type,api_ver,variant] # e.g. integration:only[create,volume,300,c7000]
+$ rake integration:only[action]                      # actions: create, update, delete
+$ rake integration:only[,type]                       # e.g. integration:only[,volume_template]
+$ rake integration:only[,,api_ver]                   # e.g. integration:only[,,300]
+$ rake integration:only[,,,variant]                  # e.g. integration:only[,,,synergy]
+$ rake integration:only[,,api_ver,variant]           # e.g. integration:only[,,300,c7000]
```

Comment:
I agree that it may be confusing, but it's documented, so it can certainly be figured out. Any way we do it will be a little confusing at first; a list of 20 different rake tasks is equally confusing in my opinion. I think it's a give-and-take, but this feels the cleanest and easiest to manage in my opinion.